pyproject.toml

```diff
@@ -203,6 +203,8 @@ select = [
   "Q",
   # pylint
   "PLE", "PLR", "PLW",
+  # ruff
+  "RUF",
```

Comment:
you just need to add
```
# tidy imports
"TID252"
```

(it would be good to check if just `"TID"` works, and if so, just use that)

Reply:
I added "TID" pyproject.toml